docs/001_develop/03_client-capabilities/008_interaction/criteria-segmented-control.mdx

```diff
@@ -0,0 +1,241 @@
+    ```html
+      <criteria-segmented-control></criteria-segmented-control>
+    ```
+    Usage
+    ```typescript
+      export function MyComponent() {
+
+        const criteriaOptions = [
+          { label: 'A', field: 'CHOSEN_OPTION', value: 'option-A', serialiser: Serialisers.EQ },
+          { label: 'B', field: 'CHOSEN_OPTION', value: 'option-B', serialiser: Serialisers.EQ },
+          { label: 'C', field: 'CHOSEN_OPTION', value: 'option-C', serialiser: Serialisers.EQ },
+        ];
+
+        const handleCriteriaChange = (event) => {
+          console.log(event.target.value)
+        }
+
+        return (
+          <criteria-segmented-control criteriaOptions={{criteriaOptions}} onClick={handleCriteriaChange}>
+            <label slot="label">Select option</label>
+          </criteria-segmented-control>
+
+        );
+      }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+      <criteria-segmented-control></criteria-segmented-control>
+    ```
+
+    Usage
+    ```typescript
+      import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+      import { FormsModule } from '@angular/forms';
+      @Component({
+        selector: 'my-root',
+        template: `
+              <criteria-segmented-control
+                [criteriaOptions]="criteriaOptions"
+                (change)="handleCriteriaChange($event)">
+                <label slot="label">Filter by instrument name</label>
+              </criteria-segmented-control>
+            `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+        imports: [FormsModule],
+      })
+      export class AppComponent {
+
+        const criteriaOptions = [
+          { label: 'A', field: 'CHOSEN_OPTION', value: 'option-A', serialiser: Serialisers.EQ },
+          { label: 'B', field: 'CHOSEN_OPTION', value: 'option-B', serialiser: Serialisers.EQ },
+          { label: 'C', field: 'CHOSEN_OPTION', value: 'option-C', serialiser: Serialisers.EQ },
+        ];
+
+        handleCriteriaChange(event: Event) {
+          const target = event.target as HTMLInputElement;
+          console.log(target.value);
+        }
+      }
+      ```
+    </TabItem>
+  </Tabs>
+
+Criteria options are JSON objecst constructed using serialisers
+
+The choice of serialisers are:
+
+```typescript
+    Serialisers.AND
+    Serialisers.OR
+    Serialisers.NOT
+    Serialisers.GT
+    Serialisers.GE
+    Serialisers.LT
+    Serialisers.LE
+    Serialisers.NE
+    Serialisers.EQ
+    Serialisers.equals
+    Serialisers.equalsIgnoreCase
+    Serialisers.contains
+    Serialisers.fieldContains
+    Serialisers.containsIgnoreCase
+    Serialisers.dateIsEqual
+    Serialisers.dateIsGreaterEqual
+    Serialisers.dateIsLessEqual
+```
+
+
+## API
+
+### Properties:
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>criteriaOptions</td>
+      <td><code>array</code></td>
```

Comment:
We can link it to the API docs when they're merged - I also notice that these types are incorrectly set to `@internal` in the package.
```suggestion
<td>

```typescript
Array<CriteriaSegmentedControlOption>
```

</td>
```

Reply:
This suggestion looks like it's incorrectly formatted on github